web/src/app/workplace/taskdetails/custom/custom-fields.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Task } from 'app/workplace/models/task';
+
+@Component({
+  selector: 'taskana-task-details-custom-fields',
+  templateUrl: './custom-fields.component.html'
+})
+export class TaskdetailsCustomFieldsComponent implements OnInit {
+
+  @Input() task: Task;
```

Comment:
This works, because I do not replace the task instance. I only modify its state. Therefore I didn't see the need to declare it as an `@output`. If you wish, I can still do it :)

Reply:
done